Bottom: 4dbaeec2062bca0ed9d4adeddb4c522812f64cd7
Top:    0fbf426c89062c0ebf519e1d95664082ac8da4a6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-13 01:59:32 +0000

Refresh of attempt-to-split-out-clip_box

---

diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index c005b26..55e88bd 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -650,7 +650,7 @@ nelma_start_png_export()
 	lastbrush = (gdImagePtr)((void *) -1);
 
 	common_set_clip_box (&nelma_graphics, &region);
-	hid_expose_callback(&nelma_graphics, 0);
+	hid_expose_callback (&nelma_graphics, 0);
 }
 
 static void
